mitmproxy/tools/console/master.py

```diff
@@ -13,7 +13,7 @@
 import threading
 
```

Comment:
It is need to use `platform.system()`

```suggestion
import platform
```